conbench/entities/benchmark_result.py

```diff
@@ -236,24 +241,65 @@ class BenchmarkResultSerializer:
 
 
 class _BenchmarkFacadeSchemaCreate(marshmallow.Schema):
-    run_id = marshmallow.fields.String(required=True)
-    run_name = marshmallow.fields.String(required=False)
-    run_reason = marshmallow.fields.String(required=False)
+    run_id = marshmallow.fields.String(
```

Comment:
@alistaire47 You might be interested + please suggest additions or corrections here if anything looks wrong